### PR TITLE
Webpack: No longer require babel to be run on .css.ts files

### PR DESCRIPTION
**Pull request by mattcompiles**

### CODE REVIEW

File: packages/integration/src/addFileScope.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    ...relative(packageInfo.dirname, filePath).split(sep),
+  );
+
+  const contents = `
+    import { setFileScope, endFileScope } from "@vanilla-extract/css/fileScope";
+    setFileScope("${normalizedPath}", ${
+    packageInfo.name ? `"${packageInfo.name}"` : 'undefined'
+  });
+    ${source}
+    endFileScope()
```
Comment (michaeltaranto) [inline packages/integration/src/addFileScope.ts:29]:
```suggestion
    setFileScope("${normalizedPath}", ${
      packageInfo.name ? `"${packageInfo.name}"` : 'undefined'
    });
    ${source}
    endFileScope();
```

**Review by michaeltaranto — APPROVED** [review]
👏

File: .changeset/fuzzy-flies-camp.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+'@vanilla-extract/webpack-plugin': minor
+---
+
+No longer require babel to be run on .css.ts files
```
Comment (markdalgleish) [inline .changeset/fuzzy-flies-camp.md:5]:
```suggestion
No longer require Babel to be run on .css.ts files
```

File: .changeset/fuzzy-flies-camp.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+Previously, the `@vanilla-extract/webpack-plugin` required the `@vanilla-extract/babel-plugin` to be run over .css.ts files. In order to bring webpack inline with the other integrations, the `@vanilla-extract/webpack-plugin` can now be used without babel.
```
Comment (markdalgleish) [inline .changeset/fuzzy-flies-camp.md:7]:
```suggestion
Previously, the `@vanilla-extract/webpack-plugin` required the `@vanilla-extract/babel-plugin` to be run over .css.ts files. In order to bring webpack inline with the other integrations, the `@vanilla-extract/webpack-plugin` can now be used without Babel. 
```

File: .changeset/fuzzy-flies-camp.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+'@vanilla-extract/webpack-plugin': minor
+---
+
+No longer require babel to be run on .css.ts files
+
+Previously, the `@vanilla-extract/webpack-plugin` required the `@vanilla-extract/babel-plugin` to be run over .css.ts files. In order to bring webpack inline with the other integrations, the `@vanilla-extract/webpack-plugin` can now be used without babel. 
+
+Note: Automatic debug ids still require the `@vanilla-extract/babel-plugin`.
```
Comment (markdalgleish) [inline .changeset/fuzzy-flies-camp.md:9]:
```suggestion
Note: Automatic debug IDs still require the `@vanilla-extract/babel-plugin`.
```

File: packages/integration/src/addFileScope.ts [hunk 5]
```diff
@@ -26,7 +26,7 @@ export function addFileScope({
     import { setFileScope, endFileScope } from "@vanilla-extract/css/fileScope";
     setFileScope("${normalizedPath}", ${packageName});
     ${source}
-    endFileScope()
+    endFileScope();
```
Comment (michaeltaranto) [inline packages/integration/src/addFileScope.ts:29]: 💪

**Pull request merged**